src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/cessations/IndividualManagingOfficerCessation.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.cessations;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.time.LocalDate;
+
+public class IndividualManagingOfficerCessation extends ManagingOfficerCessation {
+
+    @JsonProperty("officerDateOfBrith")
+    private String officerDateOfBrith;
+
+    public IndividualManagingOfficerCessation(
+            String officerAppointmentId,
+            String officerName,
+            String officerDateOfBrith,
+            LocalDate actionDate
+            ) {
+        super(officerAppointmentId, officerName, actionDate);
+        this.officerDateOfBrith = officerDateOfBrith;
+        setAppointmentType("INDIVIDUAL MO");
```

Comment:
Just going by what we have so far - could set this on the abstract class
```suggestion
        setAppointmentType("Managing Officer");
```

Reply:
Might be better to set it separately though to make it clear that it's for this MO.